packages/driver/src/cy/mouse.js

```diff
@@ -352,10 +352,14 @@ const create = (state, keyboard, focused) => {
         }
       }
 
-      if ($elements.isInput(el) || $elements.isTextarea(el) || $elements.isContentEditable(el)) {
-        if (!$elements.isNeedSingleValueChangeInputElement(el)) {
+      if (
+        $elements.isElement($elToFocus[0]) &&
+        ($elements.isInput($elToFocus[0]) || $elements.isTextarea($elToFocus[0]) || $elements.isContentEditable($elToFocus[0])) &&
+        $elements.isFocused($elToFocus[0])
+      ) {
```

Comment:
should probably pull this out into it's own function, it's a lot of follow. This means: `isTypeableElementWithFocus`?

Reply:
true, let me clean this up